content/en/account_management/saml/mobile-idp-login.md

```diff
@@ -30,6 +30,9 @@ In order to use the Datadog mobile app with Identity Provider (IdP) Initiated SA
 {{< img src="account_management/saml/datadog-mobile-idp-saml-landing-page.png" alt="Datadog Mobile SAML Interstitial" >}}
 
 ## Providers
+
+**Note:** Datadog IdP Initiated SAML should work with any identity provider. Please contact [Support][1] for help if you run into an issue. 
```

Comment:
This is a pretty vague and open statement to be adding to technical documentation ("should" and "any" make me nervous). Can we be more specific? Also, when we add a "Contact support" link like this in the docs, we need to update [this page](https://datadoghq.atlassian.net/wiki/spaces/TS/pages/328404964/The+Public+Docs+say+Contact+Support) with information for Support so that they know how to handle queries. Can you update that page to add the resources Support will need for handling?